src/Contributor/ContributorRepositoryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Contributor;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddresses;
+
+interface ContributorRepositoryInterface
```

Comment:
We used this convention of suffixing interface with `Interface` in the past, but have since switched to omitting the suffix and giving the implementations more concrete names that indicate how they are implemented. So in this case the interface should be named `ContributorRepository`, and the implementation for example `DbalContributorRepository`.

Here's a great article about it: https://verraes.net/2013/09/sensible-interfaces/